data/data/rhcos.json

```diff
@@ -0,0 +1 @@
55fca8cb", "dirty": "true", "origin": "https://github.com/coreos/coreos-assembler"}, "coreos-assembler.image-config-checksum": "b1bfc377d081a940bf2f163217888e76e5399fb290766bcc10fc009c9f6b25e9", "coreos-assembler.image-genver": "0", "coreos-assembler.image-input-checksum": "2b4449f297459803aea8643c65b2a14602082d9766c640ab5bbaa007fc9f59b6", "coreos-assembler.vm-iso-checksum": "92a1ab8f2c1ef24eadba7f84cb7e09ac911df13541eca1de239421e8f9e3adef", "images": {"openstack": {"path": "rhcos-maipo-400.7.20190306.0-openstack.qcow2.gz", "sha256": "20c8e556958d49f0baf2ef9dfbc8bc98066219a4c5fceabdfe2faa95f8ba7854", "uncompressed-sha256": "ac3acb54327724560dbf882537f02c9c9d54feaf97820325e60fed5a4c775443"}, "qemu": {"path": "rhcos-maipo-400.7.20190306.0-qemu.qcow2.gz", "sha256": "2b662fc001eb026c2ba54a98c2b167c3da3a50ddb4a833dbc6299ea1230d331d", "uncompressed-sha256": "e4ee475ec31243424f1e7bc61ef0bd9dcfe9a9817d8fa05d0e7dd072d9d3b60a"}}, "name": "rhcos-maipo", "oscontainer": {"digest": "sha256:c09f455cc09673a1a13ae7b54cc4348cda0411e06dfa79ecd0130b35d62e8670", "image": "docker-registry-default.cloud.registry.upshift.redhat.com/redhat-coreos/maipo"}, "ostree-commit": "109ec0b9b06e5070a4aec54370fa3787d8ec414d0860cb596771cc585899f2f2", "ostree-content-bytes-written": 317891942, "ostree-n-cache-hits": 16239, "ostree-n-content-total": 3093, "ostree-n-content-written": 992, "ostree-n-metadata-total": 7419, "ostree-n-metadata-written": 2338, "ostree-timestamp": "2019-03-06T22:16:26Z", "ostree-version": "400.7.20190306.0", "pkgdiff": [["atomic-openshift-clients", 2, {"NewPackage": ["atomic-openshift-clients", "4.0.17-1.git.0.2a194a0.el7", "x86_64"], "PreviousPackage": ["atomic-openshift-clients", "4.0.6-1.git.0.5dc94f3.el7", "x86_64"]}], ["atomic-openshift-hyperkube", 2, {"NewPackage": ["atomic-openshift-hyperkube", "4.0.17-1.git.0.2a194a0.el7", "x86_64"], "PreviousPackage": ["atomic-openshift-hyperkube", "4.0.6-1.git.0.5dc94f3.el7", "x86_64"]}], ["atomic-openshift-node", 2, {"NewPackage": ["atomic-openshift-node", "4.0.17-1.git.0.2a194a0.el7", "x86_64"], "PreviousPackage": ["atomic-openshift-node", "4.0.6-1.git.0.5dc94f3.el7", "x86_64"]}], ["ignition", 2, {"NewPackage": ["ignition", "0.31.0-1.gitf59a653.el7", "x86_64"], "PreviousPackage": ["ignition", "0.30.0-1.git308d7a0.el7", "x86_64"]}], ["ignition-dracut", 2, {"NewPackage": ["ignition-dracut", "0.31.0-1.gitf59a653.el7", "noarch"], "PreviousPackage": ["ignition-dracut", "0.30.0-1.git308d7a0.el7", "noarch"]}], ["pivot", 2, {"NewPackage": ["pivot", "0.0.3-3.el7", "x86_64"], "PreviousPackage": ["pivot", "0.0.3-2.el7", "x86_64"]}], ["redhat-release-coreos", 2, {"NewPackage": ["redhat-release-coreos", "4.0-20190305.0.atomic.el7.0", "x86_64"], "PreviousPackage": ["redhat-release-coreos", "4.0-20190207.0.atomic.el7.0", "x86_64"]}]], "ref": "redhat/x86_64/coreos-maipo", "rpm-ostree-inputhash": "7ab956d0beadd6e99d060b7bd6be166c0f53e14b5026ceb466e86286a5262729", "summary": "Red Hat Enterprise Linux CoreOS \"maipo\" (OpenShift)"}
```

Comment:
nit: I'd rather wash this through `jq -S` to get pretty-printing for a clearer version control.  We can also trim it, dropping `pkgdiff`, `name`, `summary`, ..., basically everything outside of `amis` and `images`.

Reply:
So we probably want a script in this repo to update it right?  `scripts/update-rhcos.sh` ?

Reply:
> So we probably want a script in this repo to update it right? `scripts/update-rhcos.sh` ?

Sure, but in `hack` (dev scripts) vs. `scripts` (user-ish scripts).  I'm also fine bumping by hand, since we will hopefully transition to #1286 before we have too many bumps via this approach.

Reply:
I added a quick `hack/update-rhcos-bootimage.py` since it was easy and having to edit the keys really demands some automation.